packages/apollo-server-errors/src/index.ts

```diff
@@ -40,6 +51,30 @@ export class ApolloError extends Error implements GraphQLError {
 
     this.extensions = { ...extensions, code };
   }
+
+  toJSON(): GraphQLFormattedError {
+    return formatError(toGraphQLError(this));
```

Comment:
We are not obsessive about coverage, but I suspect a quick test for these methods in this package would be straightforward.

Reply:
done ✅